### ISSUE TITLE
Account activation Timed out

### ISSUE THREAD

**Issue by borguleabhijeet** [codalab/codalab-competitions#2747]
I could not able to activate my account.
I have received an account confirmation **weblink which is not responsive.**
Please help me out :(

**Comment by Didayolo** [comment]
Hi,

Did you try to sign up again?

**Comment by borguleabhijeet** [comment]
Yeah, it worked. Our institution firewall preventing it to open this website.
